src/lib/metrics.ts

```diff
@@ -285,12 +285,18 @@ export default class MetricsMonitor {
             help: 'Duration of feature lifecycle stages',
         });
 
-        const stageCountByProject = createGauge({
-            name: 'stage_count_by_project',
+        const featureLifecycleStageCountByProject = createGauge({
+            name: 'feature_lifecycle_stage_count_by_project',
             help: 'Count features in a given stage by project id',
             labelNames: ['stage', 'project_id'],
         });
 
+        const featureLifecycleStageEnteredCounter = createCounter({
```

Comment:
the reason we have a counter in addition to gauges is because we want to track deltas every week